mmv1/third_party/tgc/convert.go

```diff
@@ -190,14 +198,14 @@ func (c *Converter) addDelete(rc *tfjson.ResourceChange) error {
 				} else {
 					existingConverterAsset = &asset
 				}
-				if existingConverterAsset != nil {
-					converted = converter.MergeDelete(*existingConverterAsset, converted)
-					augmented, err := c.augmentAsset(rd, c.cfg, converted)
-					if err != nil {
-						return err
-					}
-					c.assets[key] = augmented
+			}
+			if existingConverterAsset != nil {
```

Comment:
This does seem like a bug but it might be nice to fix in a separate PR for clarity.

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/pull/9790